packages/suite/src/components/wallet/WalletLayout/AccountBanners/EvmExplanationBanner.tsx

```diff
@@ -0,0 +1,92 @@
+import styled from 'styled-components';
+import { variables, Icon } from '@trezor/components';
+import { Translation } from 'src/components/suite';
+import { CloseableBanner } from './CloseableBanner';
+import { Account } from 'src/types/wallet';
+import { networks } from '@suite-common/wallet-config';
+import { useSelector } from 'src/hooks/suite/useSelector';
+import { useDispatch } from 'src/hooks/suite/useDispatch';
+import { coinsColors } from '@trezor/theme';
+import { SUITE } from 'src/actions/suite/constants';
+
+const Point = styled.div`
+    display: flex;
+    font-size: ${variables.FONT_SIZE.SMALL};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    text-align: center;
+    color: ${({ theme }) => theme.TYPE_LIGHT_GREY};
```

Comment:
we have `theme.textSubdued` 

I am also not sure if this shall be subdued at all, I think it is pretty important message. @jvaclavik WDYT?

Reply:
IMO it's fine to use `theme.textSubdued`